__tests__/app.test.js

```diff
@@ -30,4 +30,14 @@ describe("Get/api/topics", () => {
             expect(res.body.msg).toBe("Not Found");
         });
     });
-}) 
\ No newline at end of file
+})
+describe("GET /api",()=>{
+    test("responds with an object of all available API endpoints as objects",()=>{
+        return request(app)
+        .get('/api')
+        .then((response)=>{
+            expect(response.body.endpoints).toEqual(endpoints)
+            expect(response.body.endpoints).toMatchObject(endpoints)
```

Comment:
This is really good but you only need one of these and the .toMatchObject being the better option as it is more flexible.